### ISSUE TITLE
nullable DateTime isn't generating correctly

### ISSUE THREAD

**Issue by renato04** [OmarElabd/ObjectExporter#58]
Hi,

I hava a class with `DateTime?` and it is generating this way

``` cs
        Name= "Alex",
        new System.DateTime?
        {
            Date = new DateTime(2016, 10, 10, 3, 0, 0, 0),
            Day = 3,
            DayOfWeek = Monday,
            DayOfYear = 277,
            Hour = 6,
            InternalKind = 0,
            InternalTicks = 636110729996270000,
            Kind = Unspecified,
            Millisecond = 627,
            Minute = 29,
            Month = 10,
            Second = 59,
            Ticks = 636110729996270000,
            TimeOfDay = new TimeSpan(0, 6, 29, 59, 627),
            Year = 2016,
            dateData = 636110729996270000
        },
```

And i know that `DateTime` is generating correctly like this

``` cs
Date = new DateTime(2016, 10, 10, 3, 6, 57, 49),
```

Is this a bug or is this an expected behaviour?

Thanks

**Comment by OmarElabd** [comment]
Hi this was actually fixed but never made it into production ( #21 , #27 ). The fixes were included in a re-architected version of Object Exporter that i'm still working on. I'll go ahead and release some of these minor bug fixes to version 1 in the next couple of weeks.

Thanks for reporting the issue.

**Comment by OmarElabd** [comment]
Fixed
